spec/shared_examples/node_with_tx.rb

```diff
@@ -2,7 +2,7 @@
   shared_examples 'a node with properties and id' do
     describe '#neo_id' do
       it 'is a fixnum' do
-        expect(subject.neo_id).to be_a(Fixnum)
+        expect(subject.neo_id).to be_a(Integer)
```

Comment:
It can use `be_an` instead of `be_a`.

https://github.com/rspec/rspec-expectations/blob/v3.5.0/lib/rspec/matchers.rb#L361-L365